cli/wallet/wallet.go

```diff
@@ -1025,13 +1063,14 @@ func newAccountFromWIF(w io.Writer, wif string, scrypt keys.ScryptParams, label
 	return acc, nil
 }
 
-func addAccountAndSave(w *wallet.Wallet, acc *wallet.Account) error {
-	for i := range w.Accounts {
-		if w.Accounts[i].Address == acc.Address {
-			return fmt.Errorf("address '%s' is already in wallet", acc.Address)
+func addAccountAndSave(w *wallet.Wallet, acc *wallet.Account, force ...bool) error {
+	if len(force) == 0 || !force[0] {
```

Comment:
Why do you need this?

Reply:
because of https://github.com/nspcc-dev/neo-go/pull/3293#discussion_r1463551722 . it makes possible unification of adding and saving WIF/noWIF account. no-WIF approach a-priory "address is already in wallet".

Reply:
Address of multisig account must be different from single-sig address. If it's not true, then something is wrong.